packages/desktop/components/popups/CheckForUpdatesPopup.svelte

```diff
@@ -26,6 +31,7 @@
             await setAppVersionDetails()
             checkForAppUpdate()
         }
+        hasAutoUpdate = features.electron.autoUpdate[$platform]?.enabled
```

Comment:
Should we not just wrap the checkForAppUpdate with this feature flag? we also do this in the App svelte file so we need to add the feature flag there

Reply:
Ahh okay that would make sense, in that case we should direct them to the firefly website